Gemfile.lock

```diff
@@ -178,11 +179,10 @@ GEM
       mini_portile2 (~> 2.3.0)
     octokit (4.8.0)
       sawyer (~> 0.8.0, >= 0.5.3)
-    parallel (1.12.1)
-    parser (2.5.0.2)
+    parallel (1.17.0)
```

Comment:
Why did parallel update? (I'm ok with it, just curious what caused it to update.

Reply:
I just did `bundle update rubocop` and that's what it gave me. *shrug*

Reply:
I'm sure it's fine :+1: